<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuantityControl</class>
 <widget class="QDialog" name="QuantityControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/resources/background/picklit.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resources/icons/Logo2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>551</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Modify Product Quantity</string>
   </property>
   <widget class="QLabel" name="lblProductName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(238, 238, 236);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Product Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApproveQty">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/newIcons/check-icon.png</normaloff>:/resources/newIcons/check-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>114</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/newIcons/check-icon.png</normaloff>:/resources/newIcons/check-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/newIcons/missed-calls-icon.png</normaloff>:/resources/newIcons/missed-calls-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxQuantity">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>60</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProductQuantity">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblQty">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(238, 238, 236);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>QTY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteItem">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>114</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Item</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/new_icons_2/icons8-minus-96.png</normaloff>:/resources/new_icons_2/icons8-minus-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
